web/app/js/components/Tap.jsx

```diff
@@ -17,7 +17,7 @@ import { groupResourcesByNs } from './util/MetricUtils.jsx';
 import { withContext } from './util/AppContext.jsx';
 
 const urlPropsQueryConfig = {
-  autostart: { type: UrlQueryParamTypes.string }
+  autostart: StringParam,
```

Comment:
Extraneous comma after `StringParam`.

Reply:
I have seen `trailing commas` in the codebase a few times (some examples below) and I personally like them because I think they make version-control diffs cleaner. However, I haven’t seen a clear convention yet as many other times it isn’t used, so I’m happy to remove it if you prefer.  

https://github.com/linkerd/linkerd2/blob/825ad51bca68c4dd518d1b92deb91e19065c63ca/web/app/js/components/Navigation.jsx#L163-L181

https://github.com/linkerd/linkerd2/blob/5d7662fd90f6352113eb1ebcef8aa7dd2ffdaf54/web/app/js/components/ErrorBanner.jsx#L15-L34

https://github.com/linkerd/linkerd2/blob/cce6183c4ab7dbe2859819f4aaa03d15a66520d4/web/app/js/components/util/CliQueryUtils.js#L4-L8

https://github.com/linkerd/linkerd2/blob/d4f3f210ce7bed885ddf5e9513c11a2a20f31d0d/web/app/js/components/util/MetricUtils.jsx#L144-L155

Reply:
Thanks @cynthia-sg, we don't have a clear convention, as you pointed out. In this case, the original line that you replaced didn't have a trailing comma, so our preference is generally to keep it consistent to make review easier, but it's definitely a nit. 😄 

We should address this in #3667 -- if you see other inconsistencies in the code that could benefit from a formatter, please add a comment to that issue - thanks!